doc/index.rst

```diff
@@ -92,6 +92,26 @@ both ``ComputedField`` and the other field class::
 Contributing
 ------------
 
-See the `contributing docs`_.
+End Users please see the `contributing docs`_.
```

Comment:
The contributing docs are intended for all contributors, not just end users. Maybe something like "please also refer to the contributing docs in the repository
```suggestion
Please also refer to the `contributing docs`_ in the repository.
```